tests/scripts/generate_bignum_tests.py

```diff
@@ -72,6 +73,17 @@ def hex_to_int(val: str) -> int:
 def quote_str(val) -> str:
     return "\"{}\"".format(val)
 
+def combination_pairs(values: List[T]) -> List[Tuple[T, T]]:
+    """Return all pair combinations from input values.
+
+    The return value is cast, as older versions of mypy are unable to derive
+    the specific type returned by itertools.combinations_with_replacement.
```

Comment:
Non-blocker: this information is relevant when reading the function's code, but not when using the function. So it should be a comment, not part of the documentation string.

Reply:
https://github.com/Mbed-TLS/mbedtls/pull/6296 has fixes for this and the other non-blocker issues I raised in this review.